storefront/modules/catalog/services/ProductService.ts

```diff
@@ -4,53 +4,55 @@ import { ProductAll, ProductFeature } from '../models/ProductFeature';
 import { ProductOptionValueGet } from '../models/ProductOptionValueGet';
 import { ProductVariation } from '../models/ProductVariation';
 import { ProductsGet } from '../models/ProductsGet';
+import apiClientService from '@/common/services/ApiClientService';
 
 export async function getFeaturedProducts(pageNo: number): Promise<ProductFeature> {
-  const response = await fetch(`api/product/storefront/products/featured?pageNo=${pageNo}`);
+  const response = await apiClientService.get(
+    `api/product/storefront/products/featured?pageNo=${pageNo}`
+  );
   return response.json();
 }
 
 export async function getProductDetail(slug: string): Promise<ProductDetail> {
-  const response = await fetch(process.env.API_BASE_PATH + '/product/storefront/product/' + slug);
+  const response = await apiClientService.get(
+    process.env.API_BASE_PATH + '/product/storefront/product/' + slug
+  );
   return response.json();
 }
 
 export async function getProductOptionValues(productId: number): Promise<ProductOptionValueGet[]> {
-  const res = await fetch(
+  const res = await apiClientService.get(
     `${process.env.API_BASE_PATH}/product/storefront/product-option-combinations/${productId}/values`
   );
   if (res.status >= 200 && res.status < 300) return res.json();
-  return Promise.reject(res);
+  throw res;
 }
 
 export async function getProductByMultiParams(queryString: string): Promise<ProductAll> {
-  const res = await fetch(`/api/product/storefront/products?${queryString}`);
+  const res = await apiClientService.get(`/api/product/storefront/products?${queryString}`);
```

Comment:
Please be consistent in handling URLs:
#L18:
`process.env.API_BASE_PATH + '/product/storefront/product/' + slug`
#L32:
`/api/product/storefront/products?${queryString}`

Reply:
I've reformatted and use #L32 as the common way to handle URLs